ingest/ingestor.py

```diff
@@ -0,0 +1,19 @@
+import abc
+
+
+class Ingestor(metaclass=abc.ABCMeta):
+    """
+    The command interface that declares a method (execute) for a particular
+    action.
+    """
+    @abc.abstractmethod
+    def execute_ingest(self, ):
+        pass
+
+    """
+    Returns true/false for whether this ingestor can handle the given file type
+    type should correspond to the
+    """
```

Comment:
move comment to inside method and finish the sentence, per Eric's comment. (I think this might have been code from my merge -- sorry)

Reply:
This file wasn't suppose to be included. 12c83ea